rust/relay/src/server.rs

```diff
@@ -324,11 +360,11 @@ where
     #[tracing::instrument(level = "debug", skip_all, fields(%sender, %allocation, recipient, channel))]
     pub fn handle_peer_traffic(
         &mut self,
-        bytes: &[u8],
+        mut msg: PeerToClient,
```

Comment:
I'm seeing `ClientToPeer` and `PeerToClient`.

This is for the relay, right? The Client is not a peer of the relay? Is Peer a Gateway?

Reply:
In the TURN protocol, a client is the one talking to port 3478, making allocations and channel bindings. A peer is another node talking to an allocated port. The peer has no idea that it is talking to a relay because it just sends UDP traffic.

You can think of an allocation like a "remote socket". It is as if you would call `UdpSocket::bind` on one of the interfaces of the relay. Peers talk to the network end of the socket. Clients are the program and use the API to talk to it (in the form of `ChannelData` messages).